sonar-project.properties

```diff
@@ -1,5 +1,5 @@
-sonar.projectKey=hyperledger_aries-cloudagent-python
-sonar.organization=hyperledger
-sonar.projectName=aries-cloudagent-python
+sonar.projectKey=penwallet-foundation_acapy
```

Comment:
I'd revert this change. It's covered in my PR. If you do include it please fix the spelling mistake.

Reply:
Doh!! Should have known better.  Reverted.